fdbserver/workloads/RemoveServersSafely.actor.cpp

```diff
@@ -296,13 +296,14 @@ struct RemoveServersSafelyWorkload : TestWorkload {
 		TraceEvent("RemoveAndKill").detail("Step", "exclude list first").detail("ToKill", describe(toKill1)).detail("KillTotal", toKill1.size()).detail("ClusterAvailable", g_simulator.isAvailable());
 		self->excludeAddresses(toKill1);
 
-		Optional<Void> result = wait( timeout( removeAndKill( self, cx, toKill1, NULL), self->kill1Timeout ) );
+		Optional<Void> result = wait( timeout( removeAndKill( self, cx, toKill1, NULL, false), self->kill1Timeout ) );
 
 		bClearedFirst = result.present();
-
+		// killProcArray is always empty here so why are we tracing it? is it meant to be something else or is a step missing somewhere?
 		TraceEvent("RemoveAndKill").detail("Step", "excluded list first").detail("Excluderesult", bClearedFirst ? "succeeded" : "failed").detail("KillTotal", toKill1.size()).detail("Processes", killProcArray.size()).detail("ToKill1", describe(toKill1)).detail("ClusterAvailable", g_simulator.isAvailable());
 
-		bClearedFirst=false;
+		// this is never unset after this, is this line supposed to be here? below conditionals could all just be hard-coded instead if intentional
```

Comment:
PTAL

Reply:
What if you comment out ` bClearedFirst=false;` in master, instead of this PR? 
See if commenting out  ` bClearedFirst=false;` can break master.

Theoretically, I think `bClearedFirst` can be randomly set `true` or `false`. It simulates DBA's different behaviors of removing an address.

Reply:
Yes, I agree with the idea that it can be either one, which is why I think this line should be removed, since it forces only one possibility.